pytest_sftpserver/core.py

```diff
@@ -0,0 +1,19 @@
+from pytest_sftpserver.sftp.server import SFTPServer
+
+
+class SFTPServerContextDecorator(object):
```

Comment:
For my taste this name is a bit too `how` and not enough `what` to me.

Some ideas for more descriptive names:
- `sftpserver_factory`
- `provide_sftpserver`
- `inject_sftpserver`